### PR TITLE
Update Disable-CsOnlineDialInConferencingUser.md

### PR DESCRIPTION
**Pull request by ruizoscar**
Adding a note to outline the deprecation of this cmdlet.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Disable-CsOnlineDialInConferencingUser.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ ms.reviewer:
 # Disable-CsOnlineDialInConferencingUser
 
 ## SYNOPSIS
+
+> [!NOTE]
+> This cmdlet will be deprecated on December 31, 2021. To disable Audio Conferencing for a given user, unassign the Audio Conferencing license from the user or disable the Audio Conferencing component from the user's license. For additional information, see [Assign or remove licenses for Microsoft 365 Apps for business](https://docs.microsoft.com/en-us/microsoft-365/admin/add-users/add-users).
```
Comment (dariomws) [inline skype/skype-ps/skype/Disable-CsOnlineDialInConferencingUser.md:18]:
```suggestion
> This cmdlet will be deprecated on December 31, 2021. To disable Audio Conferencing for a given user, unassign the Audio Conferencing license from the user or disable the Audio Conferencing component from the user's license. For additional information, see [Assign or remove licenses for Microsoft 365 Apps for business](https://docs.microsoft.com/microsoft-365/admin/add-users/add-users).
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Maybe the article could be replaced by:
https://docs.microsoft.com/microsoftteams/manage-the-audio-conferencing-settings-for-my-organization-in-teams#assign-an-audio-conferencing-license

**Review by get-itips — APPROVED** [review]
I am ok after suggestion is applied

**Review by tseward — APPROVED** [review]
Also good after changes are applied.

**Comment by yogkumgit** [comment]
@ruizoscar Please implement the suggested changes. Thanks

**Comment by ruizoscar** [comment]
Hi @yogkumgit, I don't have access to publish the changes or merge this PR. Would it be possible for you or someone from the editorial team to publish these changes?

**Comment by tiburd** [comment]
close > reopen

**Pull request merged**